src/symbols.jl

```diff
@@ -1,344 +1,2 @@
-    "gtrsim" => '≳',
-    "gtrless" => '≷',
-    "gtreqless" => '⋛',
-    "gtrdot" => '⋗',
-    "gimel" => 'ℷ',
-    "ggg" => '⋙',
-    "gg" => '≫',
-    "geqq" => '≧',
-    "geq" => '≥',
-    "ge" => '≥',
-    "Gamma" => 'Γ',
-    "gamma" => 'γ',
-    "frown" => '⌢',
-    "forall" => '∀',
-    "flat" => '♭',
-    "Finv" => 'Ⅎ',
-    "fallingdotseq" => '≒',
-    "exists" => '∃',
-    "eth" => 'ð',
-    "eta" => 'η',
-    "equiv" => '≡',
-    "eqcirc" => '≖',
-    "epsilon" => '∊',
-    "emptyset" => '∅',
-    "ell" => 'ℓ',
-    "downharpoonright" => '⇂',
-    "downharpoonleft" => '⇃',
-    "downdownarrows" => '⇊',
-    "Downarrow" => '⇓',
-    "downarrow" => '↓',
-    "dots" => '…',
-    "dotplus" => '∔',
-    "doteqdot" => '≑',
-    "doteq" => '≐',
-    "divideontimes" => '⋇',
-    "div" => '÷',
-    "digamma" => 'Ϝ',
-    "diamondsuit" => '♢',
-    "Diamond" => '◇',
-    "diamond" => '⋄',
-    "Delta" => 'Δ',
-    "delta" => 'δ',
-    "ddots" => '⋱',
-    "ddagger" => '‡',
-    "dashv" => '⊣',
-    "dashrightarrow" => '⇢',
-    "dashleftarrow" => '⇠',
-    "daleth" => 'ℸ',
-    "dagger" => '†',
-    "curvearrowright" => '↷',
-    "curvearrowleft" => '↶',
-    "curlywedge" => '⋏',
-    "curlyvee" => '⋎',
-    "curlyeqsucc" => '⋟',
-    "curlyeqprec" => '⋞',
-    "Cup" => '⋓',
-    "cup" => '∪',
-    "coprod" => '∐',
-    "cong" => '≅',
-    "complement" => '∁',
-    "clubsuit" => '♣',
-    "circledS" => 'Ⓢ',
-    "circleddash" => '⊝',
-    "circledcirc" => '⊚',
-    "circledast" => '⊛',
-    "circlearrowright" => '↻',
-    "circlearrowleft" => '↺',
-    "circeq" => '≗',
-    "circ" => '∘',
-    "chi" => 'χ',
-    "centerdot" => '⋅',
-    "cdots" => '⋯',
-    "cdot" => '⋅',
-    "cbrt" => '∛',
-    "Cap" => '⋒',
-    "cap" => '∩',
-    "Bumpeq" => '≎',
-    "bumpeq" => '≏',
-    "bullet" => '∙',
-    "boxtimes" => '⊠',
-    "boxplus" => '⊞',
-    "boxminus" => '⊟',
-    "boxdot" => '⊡',
-    "Box" => '□',
-    "bowtie" => '⋈',
-    "bot" => '⊥',
-    "blacktriangleright" => '▶',
-    "blacktriangleleft" => '◀',
-    "blacktriangledown" => '▼',
-    "blacktriangle" => '▲',
-    "blacksquare" => '■',
-    "blacklozenge" => '◆',
-    "bigwedge" => '⋀',
-    "bigvee" => '⋁',
-    "biguplus" => '⨄',
-    "bigtriangleup" => '△',
-    "bigtriangledown" => '▽',
-    "bigstar" => '★',
-    "bigsqcup" => '⨆',
-    "bigotimes" => '⨂',
-    "bigoplus" => '⨁',
-    "bigodot" => '⨀',
-    "bigcup" => '⋃',
-    "bigcirc" => '○',
-    "bigcap" => '⋂',
-    "between" => '≬',
-    "beth" => 'ℶ',
-    "beta" => 'β',
-    "because" => '∵',
-    "barwedge" => '⊼',
-    "backsim" => '∽',
-    "backprime" => '‵',
-    "backepsilon" => '∍',
-    "asymp" => '≍',
-    "ast" => '∗',
-    "approxeq" => '≊',
-    "approx" => '≈',
-    "angle" => '∠',
-    "alpha" => 'α',
-    "aleph" => 'ℵ'
-)
+# Here, we collect and reverse the REPL's latex autocompletion map.
+latex_symbol_map = REPL.symbols_latex |> collect .|> (x -> (x[2][2:end] => x[1])) |> x -> Dict(x...)
```

Comment:
`REPL.symbols_latex` seems to be empty! This one works:
### ```suggestion
using REPL
const latex_symbol_map = Dict((k => v[1] for (k, v) in REPL.REPLCompletions.latex_symbols))
```

Reply:
This of course also needs REPL in the project...

Reply:
Huh, it worked on my machine - wonder why.  Will add it to the project.

Reply:
Btw, I must say I disklike the usage of `.|>` and anon functions like that :D 
I find it very hard to read and the stack traces become hard to read, so I'd really like to avoid this in my Packages ;)

Reply:
Yeah, will try to use them less - I essentially copy-pasted this from messing around in the REPL :D

Reply:
Fair enough^^ Well, my suggestion should work ;)